resources/css/app.css

```diff
@@ -6,5 +6,5 @@
     @apply inline-block transition border-b;
 }
 .border-hover:hover {
```

Comment:
Can you set your global `.gitignore` to exclude `.DS_Store` files?

https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer